pkg/expr/sql/parser_allow.go

```diff
@@ -138,6 +138,31 @@ func allowedNode(node sqlparser.SQLNode) (b bool) {
 	case *sqlparser.Where:
 		return
 
+	// // New safe node types
+	// case sqlparser.Values, sqlparser.ValTuple:
+	// 	return
+
+	// case *sqlparser.GroupConcatExpr:
+	// 	return
+
+	// case *sqlparser.Default:
+	// 	return
+
+	// case *sqlparser.NullVal:
+	// 	return
+
+	// case *sqlparser.MatchExpr:
+	// 	return
+
+	// case *sqlparser.CollateExpr:
+	// 	return
+
+	// case *sqlparser.WindowDef:
+	// 	return
+
+	// case *sqlparser.Frame, *sqlparser.FrameExtent, *sqlparser.FrameBound:
```

Comment:
We don't need to explicitly allow Frame because:

- It's a child node of Over, which we already allow
- The parser treats it as part of the window specification structure
- All its components (like OrderBy and Exprs) are already allowed